ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml

```diff
@@ -13,6 +13,7 @@ spec:
     - name: git_email
       default: "no-reply@redhat.com"
     - name: upstream_repo_name
+      default: ""
```

Comment:
@Allda @MarcinGinszt can you think of a better way to default or generate this value? It's only used when submit="true".